docker/api/Dockerfile

```diff
@@ -46,10 +46,12 @@ RUN chown -R www-data:www-data /tmp/devicecerts && \
 # Give permission for unittests
 RUN chown root:www-data /opt/cnaas/*.sh && \
     chmod g+rx /opt/cnaas/*.sh
-RUN chown -R www-data:www-data /opt/cnaas/venv/cnaas-nms/src/
+RUN chown -R www-data:www-data /opt/cnaas/venv/
 
 # Branch specific, don't cache
 ADD "https://api.github.com/repos/SUNET/cnaas-nms/git/refs/heads/" latest_commit
+
+USER www-data
```

Comment:
This is what actually solves the problem right?

Reply:
After some hours of vigorous testing and multiple docker-compose build runs, I am convinced that for this patch to work, all changes are needed.

- user must be www-data
- venv must be activated when running pip install in `cnaas-setup-branch.sh`. Otherwise, requirements are installed in ~SOME PLACE~ (= the user's home dir) and never used by uwsgi.
- for this, www-data needs write access on the venv

It's the most minimally invasive quickfix for #181 that I can come up with right now, without refactoring the whole API Dockerfile (which is another task … for another day).